### ISSUE TITLE
no such file or directory, open 'target/scratch/tree-sitter.js' when installing HEAD via homebrew on mac

### ISSUE THREAD

**Issue by kikito** [tree-sitter/tree-sitter#1217]
I tried this command on Mac OS:

```
brew install tree-sitter --HEAD
```
And got this log:
```
==> Cloning https://github.com/tree-sitter/tree-sitter.git
Updating /Users/kikito/Library/Caches/Homebrew/tree-sitter--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at 2b83500b Merge pull request #1208 from CyberShadow/pull-20210626-181203
==> make
==> make install PREFIX=/usr/local/Cellar/tree-sitter/HEAD-2b83500
==> npm install -ddd --build-from-source --cache=/Users/kikito/Library/Caches/Homebrew/npm_cache
==> script/build-wasm
Last 15 lines from /Users/kikito/Library/Logs/Homebrew/tree-sitter/04.build-wasm:

This command will now exit. When you are done editing those paths, re-run it.
==============================================================================

ERROR: ENOENT: no such file or directory, open 'target/scratch/tree-sitter.js'
    at Object.openSync (node:fs:582:3)
    at Object.readFileSync (node:fs:450:35)
    at read_file (/private/tmp/tree-sitter-20210629-92686-48phc0/lib/binding_web/node_modules/terser/bin/uglifyjs:349:19)
    at /private/tmp/tree-sitter-20210629-92686-48phc0/lib/binding_web/node_modules/terser/bin/uglifyjs:176:37
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/private/tmp/tree-sitter-20210629-92686-48phc0/lib/binding_web/node_modules/terser/bin/uglifyjs:175:28)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)

READ THIS: https://docs.brew.sh/Troubleshooting

Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels.
```

Other info:

```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.14.6
BuildVersion:	18G5033

$ node --version 
v16.4.0

$ brew --version
Homebrew 3.2.0-83-g04532cb
Homebrew/homebrew-core (git revision 080debca51; last commit 2021-06-29)
Homebrew/homebrew-cask (git revision b6cbd7a144; last commit 2021-06-29)
```

**Comment by carlocab** [comment]
`brew install --HEAD tree-sitter` worked for me about half an hour ago, but I'm on (Intel) Big Sur.

If you need this to do `brew install --HEAD neovim`, I suggest you wait a little instead. That should be resolved by one of #1216 or neovim/neovim#14930.

**Comment by kikito** [comment]
Fixed by #1216 

Thanks for the feedback, @carlocab !
